cookiecutter.json

```diff
@@ -7,5 +7,6 @@
 	"project_short_description": "A flasky app.",
 	"use_pipenv": ["no", "yes"],
 	"python_version": ["3.7", "3.6"],
-	"node_version": ["12", "10", "8"]
+	"node_version": ["12", "10", "8"],
+	"deployment_on_heroku": ["no", "yes"]
```

Comment:
Instead of adding a step for Heroku specifically, what about prompting the user to select their preferred deploy option? For now, the only two options would be "Manual" (default) or "Heroku"--but if additional options are added in the future, they could be wrapped into this single prompt, rather than needing new steps for each option.

Reply:
I was thinking about this and my proposition is to simplify it by asking just one question: `use_heroku`. I've add rewrote little bit informations about Heroku and deployment in README. I think the question about manual or automatic deployment can cause unnecessary confusion and add avoidable complexity.